com.obsproject.Studio.Plugin.Backgroundremoval.yaml

```diff
@@ -0,0 +1,55 @@
+id: com.obsproject.Studio.Plugin.Backgroundremoval
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.4
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/Backgroundremoval
+cleanup:
+  - /include
+  - /share/opencv4
+  - /lib/cmake
+  - /bin
+modules:
+  # onnxruntime has a bash+python+cmake build script - just download/use pre-compiled library
+  # https://onnxruntime.ai/docs/build/inferencing.html
+  - name: onnxruntime
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/microsoft/onnxruntime/releases/download/v1.14.1/onnxruntime-linux-x64-1.14.1.tgz
+        sha256: 010585f534d822bf02d54c7cfc53b57aa3eb0cf34f8fb7503ee519ed18e25d29
```

Comment:
Have you considered adding extra metadata so that flatpak-external-data-checker can automatically keep the dependencies updated for your? You can read more here: https://github.com/flathub/flatpak-external-data-checker#changes-to-flatpak-manifests